src/types/ScreenInfoUiaProviderBase.cpp

```diff
@@ -381,30 +347,26 @@ IFACEMETHODIMP ScreenInfoUiaProviderBase::GetVisibleRanges(_Outptr_result_mayben
             return hr;
         }
     }
+    UiaTracing::TextProvider::GetVisibleRanges(*this, *range.Get());

```

Comment:
range might be null, which would make this explode _spectacularly_.. right/

Reply:
oh i guess not... if you ever got out here with it being null..
hmm\
okay. but i don't love it. comment?

Reply:
also, wait, no. you're calling `range.Detach()`. That is definitely not going to be an okay ComPtr to call `.Get` on later.

Reply:
You're absolutely right. Unfortunately, this entire function is just all sorts of wrong (see #4507).

Commenting the tracing off of this function and adding a TODO work item. 😡